app/dashboard/models.py

```diff
@@ -658,12 +658,30 @@ def action_urls(self):
 
         """
         return {
-            'fulfill': f"/issue/fulfill/{self.pk}",
+            'fulfill': f"/issue/fulfill?pk={self.pk}&network={self.network}",
             'increase': f"/issue/increase/{self.pk}",
```

Comment:
is it worth making all of these URLs follow the same format?  i know its a little bit of scope creep but the consistency will be worth it i think